debian/changelog

```diff
@@ -1,3 +1,13 @@
+pg-auto-failover-enterprise (1.6.2-1) stable; urgency=low
+  
+  This release is a bugfix release on-top of the community version.
```

Comment:
This line does not conform to the debian changelog specs. 
See details at https://www.debian.org/doc/debian-policy/ch-source.html#s-dpkgchangelog

```
package (version) distribution(s); urgency=urgency
  [optional blank line(s), stripped]
  * change details
  more change details
  [blank line(s), included in output of dpkg-parsechangelog]
  * even more change details
  [optional blank line(s), stripped]
 -- maintainer name <email address>[two spaces]  date
```

Reply:
@gurkanindibay Ping

Reply:
Fixed